src/css/footer.css

```diff
@@ -181,42 +182,41 @@ section#links-footer {
 }
 
 /* Media queries */
-@media (max-width: 1279px) {
-
+@media (width <= 1279px) {
     section#links-footer {
-        padding: 50px;
+        padding: 3.125rem;
     }
 
     section#links-footer .section-container {
         display: flex;
         flex-direction: column;
-        gap: 50px;
+        gap: 3.125rem;
         box-sizing: border-box;
         overflow-x: hidden;
     }
 
-    #links-footer .section-container > div {
+    section#links-footer .section-container>div {
         width: 100%;
         max-width: 100%;
         border: none !important;
         border-left: none !important;
-        border-bottom: 2px solid #0000003b !important;
+        border-bottom: 0.125rem solid #0000003b !important;
```

Comment:
Ik zie hier een paar keer `!important` staan. Mijn rule-of-thumb is. Ik mag geen `!important` gebruiken, dit laat zien dat er niet slim gebruik is gemaakt van selectors. Het enige geval zou begrijpelijk zijn, als je gebruik maakt van een componenten library, maar die gebruik je nu niet.